engine/templates_tests/execScript.yaml

```diff
@@ -0,0 +1,18 @@
+name: exec-script
+description: Executing a python script
+title_format: "[test] a simple task"
+
+inputs:
+- name: file
+  description: file path
+
+steps:
+    stepOne:
+        description: first step
+        action:
+            type: script
+            configuration:
+                file: "{{.input.file}}"
```

Comment:
we shouldn't allow this.
A template should be quite determinist, and this use case sounds like a "meta-template" and is a possible vulnerability.

Reply:
This is only a test template used for internal tests, not an example.

Reply:
can you confirm that we don't support this feature ?

Reply:
right now it works (naturally), even though it's dumb.
see above discussion around **ValidConfig()** suggestion.